tasks/preflight.yml

```diff
@@ -96,9 +96,26 @@
   run_once: true
 
 - block:
+    - name: Setup variable with GH_TOKEN
+      set_fact:
+        _gh_token: "{{ lookup('env', 'GH_TOKEN') | default('') }}"
+      delegate_to: localhost
```

Comment:
This part may not be working like I expected. But curious if you think this is the right direction.